Modules/Sources/JetpackStats/Views/LegacyFloatingDateControl.swift

```diff
@@ -159,6 +160,24 @@ private struct StatsDateRangeTip: Tip {
     }
 }
 
+private struct MinimumBottomSafeArea: ViewModifier {
```

Comment:
It solves the problem described here: https://www.hackingwithswift.com/forums/swiftui/adding-bottom-spacing-on-non-face-id-phones/12181